docs/modules/ROOT/pages/getting-started/relationships-and-advanced-filtering.adoc

```diff
@@ -114,81 +115,85 @@ WHERE (
     )
 ```
 
-This complex filter is actually composed of 3 basic comparisons:
+This advanced filter is composed of 3 basic comparisons:
 
 * `a.name = "Keanu Reeves"`
 * `m.title CONTAINS "Matrix"`
 * `m.released < 2000`
 
-Then, these are composed of the logic operations `AND`, `NOT`, and `OR`.
+These comparisons are built around the logic operations `AND`, `NOT`, and `OR`, respectivelly.
```

Comment:
This is not completely accurate, as these comparisons are not built around the logic. Instead, we are going to use the logic `AND`, `NOT` and `OR` on these comparisons

My previous text was a bit ambiguous on that, sorry :sweat_smile: